tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -68,6 +68,14 @@ sub run {
 
 }
 
+sub sol_login {
```

Comment:
How about moving it to [ipmi_backend_utils.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/ipmi_backend_utils.pm)?

Reply:
Yes, good suggestion. The functionis expected to be used in yy vgpu test as well.